pre_processing/read_seviri.F90

```diff
@@ -485,8 +510,11 @@ subroutine read_seviri_l1_5_nat_or_hrit(l1_5_file, imager_geolocation, &
       imager_angles%solzen(startx:,:,1)       = preproc%sza(column0+1:column1+1,line0+1:line1+1)
       imager_angles%solazi(startx:,:,1)       = preproc%saa(column0+1:column1+1,line0+1:line1+1)
       imager_angles%satzen(startx:,:,1)       = preproc%vza(column0+1:column1+1,line0+1:line1+1)
-      imager_angles%satazi(startx:,:,1)       = preproc%vaa(column0+1:column1+1,line0+1:line1+1)
+      !imager_angles%satazi(startx:,:,1)       = preproc%vaa(column0+1:column1+1,line0+1:line1+1)
       imager_measurements%data(startx:,:,:)   = preproc%data(column0+1:column1+1,line0+1:line1+1,:)
+
+      ! Offset subset for inversion of satazi
+      imager_angles%satazi(startx:,:,1)       = preproc%vaa(3712-column1:3712-column0,line0+1:line1+1)
```

Comment:
This is hard-coded for full disk imagery (3712 columns), and will fail for super rapid scan mode.

Reply:
True

Reply:
Can you use the image sizes from `read_seviri_dimensions` instead of hardcoding `3712`? From memory that should contain the correct size.

Reply:
I'm making that change but `read_seviri_dimensions` just returns 3712.